felix/dataplane/linux/int_dataplane.go

```diff
@@ -434,9 +435,16 @@ func NewIntDataplaneDriver(config Config) *InternalDataplane {
 	dp.ipSets = append(dp.ipSets, ipSetsV4)
 
 	if config.RulesConfig.VXLANEnabled {
-		routeTableVXLAN := routetable.New([]string{"^vxlan.calico$"}, 4, true, config.NetlinkTimeout,
-			config.DeviceRouteSourceAddress, config.DeviceRouteProtocol, true, 0,
-			dp.loopSummarizer)
+		var routeTableVXLAN routeTable
+		if !config.RouteSyncDisabled {
+			log.Info("RouteSyncDisabled is false.")
```

Comment:
In most cases `RouteSyncDisabled` will be false so I suggest to use `log.Debug` . Similar suggestions apply to other RouteTable below.

Reply:
Sounds good, I'll change it for a `log.Debug` when we hit the default value for `RouteSyncDisabled`